src/app/pages/news/9.4/11-tracing-system-overview.md

```diff
@@ -1,16 +1,22 @@
-## Request Tracing and System Overview {#tracing}
+## Slow Requests and Traffic Graph {#tracing}
 
-User experience is a top priority in process automation projects. 
-That is why the Axon Ivy Portal has been massively enhanced.
+It is key for an orchestration platform like Axon Ivy to provide tools to analyze problems. 
+The new Slow Requests and Traffic Graph views in the Engine Cockpit provides 
```

Comment:
provides --> provide (views: plural, not singular)

all information --> all the information

to find the system in your orchestrated solution that makes trouble. -->
to identify the source of trouble in your orchestrated solution.